ingestion/src/metadata/utils/sql_queries.py

```diff
@@ -536,3 +536,55 @@
   from SVV_TABLE_INFO
   where diststyle not like 'AUTO%%'
 """
+
+DAGSTER_PIPELINE_DETAILS_GRAPHQL = """
```

Comment:
Technically these are not sql queries so it should be in these file, maybe you can create a file like `graphql_queries.py` and move it there

Reply:
Ok 👍